js/TabsView.js

```diff
@@ -13,7 +13,7 @@ class TabsView extends ComponentView {
   }
 
   get layout() {
-    const layout = (device.screenSize === 'large')
+    const layout = (device.screenSize === 'large' || device.screenSize === 'xlarge')
```

Comment:
```suggestion
    const layout = device.isScreenSizeMin('large')
```

Use [isScreenSizeMin](https://github.com/adaptlearning/adapt-contrib-core/blob/2cdc958e2841e7915683782b031704dfedd61684/js/device.js#L74-L82)